popup.js

```diff
@@ -5,6 +5,7 @@ chrome.tabs.query({ active: true, currentWindow: true }, tabs => {
   const url = tabs[0].url;
   chrome.storage.local.get("scroll-mark", data => {
     const scrollMarkData = data["scroll-mark"];
+    const scrollMarkOffset = scrollMarkData.offset;
```

Comment:
Where is this being used in this file?

Reply:
Yeah, it wasn't being used. Pushed all the changes.